src/components/Menu/Menu.tsx

```diff
@@ -194,12 +194,21 @@ export const Menu: ChakraComponent<
             return <Icon name={media.name} size="medium" />;
           } else {
             return (
-              <Image
-                src={media.src}
-                alt={media.alt}
-                size="xxxsmall"
-                sx={{ borderRadius: "24px" }}
-              />
+              <Box w="1.5rem" h="1.5rem">
+                <Image
+                  alt={media.alt}
+                  src={media.src}
+                  /** Using custom styles to handle the size and rounded shape
+                   * because the complexity if the Image component is not
```

Comment:
nit typo "if".

I wonder when this broke. I don't recall the production `Menu` image items being skewed the way the are now.

Reply:
The images have been oddly shaped as long as I can remember.